capabilities.go

```diff
@@ -30,6 +30,32 @@ func (c Capabilities) Browser(name string) Capabilities {
 	return c
 }
 
+// A ProxyConfig instance defines the desired proxy configuration the WebDriver
+// should use to proxy a Page.
+//
+//    ProxyType is required and defines the type of proxy being used
+//    Possible Values:
+//        {direct|manual|pac|autodetect|system}
+//
+// See: https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities#proxy-json-object
+type ProxyConfig struct {
```

Comment:
Do any of these fields actually need to be provided if blank?
Can we use `string`s and `omitempty` instead of `*string`s?